style.css

```diff
@@ -130,11 +131,14 @@ u {
 button,
 input[type="submit"],
 input[type="reset"] {
+  all: unset;
```

Comment:
Is this causing the text alignment to be off? https://98css-ntl6d9tp5-jdan.vercel.app/